tool/site/client/src/services/test-packs/xls.ts

```diff
@@ -215,6 +215,7 @@ export interface PrescriptionRow {
   issueDurationInDays: string
   dispenserNotes: Array<string>
   nominatedPharmacy?: string
+  nominatedPharmacyType: string
```

Comment:
Hey @MathewArnoldIW,  Is this correct that the nominatedPharmacyType can't be nullable? It looks like the nominatedPharmacy can be null. I'm thinking you cant have a nominatedPharmacyType if you haven't got a nominatedPharmacy? 

give me a shout if you want to discuss any of this?

Reply:
Discussed and agreed with Andy